packages/tenant-readmodel-writer/test/converterV1.ts

```diff
@@ -35,6 +35,14 @@ export function toFeatureV1(feature: TenantFeature): TenantFeatureV1 {
         },
       },
     }))
+    .with({ type: "DelegatedProducer" }, (feature) => ({
+      sealedValue: {
+        oneofKind: undefined,
+        producer: {
+          availabilityTimestamp: feature.availabilityTimestamp,
+        },
+      },
+    }))
```

Comment:
Is this really needed? The V1 tenant has no `DelegatedProducer` feature

Reply:
In detail what this function should returns in other case? 
Which yourexpected behaviour in `DelegatedProducer` case?
The `TenantFeature` type is a discriminated union  and `match` method require to handle both. Another way, more clear, is to throw an exception when try to convert into `TenantFeatureV1` the `DelegatedProducer` feature.

Reply:
this [commit](https://github.com/pagopa/interop-be-monorepo/pull/1104/commits/5efb8d880e1414e82fee0f7dbe348864bdfce47c)  improve this method.